.buildkite/scripts/build_kibana.sh

```diff
@@ -9,17 +9,11 @@ export KBN_NP_PLUGINS_BUILT=true
 echo "--- Build Kibana Distribution"
 
 BUILD_ARGS=""
-if is_pr_with_label "ci:build-all-platforms"; then
-  BUILD_ARGS="--all-platforms --skip-os-packages"
-fi
-if is_pr_with_label "ci:build-os-packages"; then
-  BUILD_ARGS="--all-platforms --docker-cross-compile"
-fi
-if ! is_pr_with_label "ci:build-canvas-shareable-runtime"; then
-  BUILD_ARGS="$BUILD_ARGS --skip-canvas-shareable-runtime"
-fi
-
-node scripts/build "$BUILD_ARGS"
+is_pr_with_label "ci:build-all-platforms" && BUILD_ARGS="--all-platforms --skip-os-packages"
+is_pr_with_label "ci:build-os-packages" && BUILD_ARGS="--all-platforms --docker-cross-compile"
+is_pr_with_label "ci:build-canvas-shareable-runtime" || BUILD_ARGS="$BUILD_ARGS --skip-canvas-shareable-runtime"
+is_pr_with_label "ci:build-docker-contexts" || BUILD_ARGS="$BUILD_ARGS --skip-docker-contexts"
+node scripts/build $(echo "$BUILD_ARGS")
```

Comment:
I feel like the actual issue with the previous version was that it was wrapped in `""`, have you tried just removing them? Should have the same effect as this, right?